west.yml

```diff
@@ -136,6 +138,12 @@ manifest:
       path: modules/fs/fatfs
       groups:
         - fs
+    - name: hal_adi
+      remote: analogdevicesinc
```

Comment:
Can remove this now that https://github.com/zephyrproject-rtos/hal_adi  has been created